release/cloud_build.template.json

```diff
@@ -7,7 +7,7 @@
           "path": "/output"
 	}
       ],
-      "name": "gcr.io/istio-io/istio-builder:0.4.4",
+      "name": "gcr.io/istio-testing/istio-builder:0.4.5-bazel7-go19",
```

Comment:
Why bazel ?

Reply:
It's just an existing image I found that was already made from November. Normally these just have x.y.z as the tag but on this one I think Seb tried to be more descriptive about the contents.  Longer term an image from istio-testing will need to be pushed to the usual location in istio-io (either with this tag or a shorter 0.4.x tag), but for now this was a way to get go 1.9 (and time time I ran a build the push scripts were still using bazel).